ludwig/schema/features/loss/__init__.py

```diff
@@ -0,0 +1 @@
+from ludwig.schema.features.loss.loss import get_loss_classes, get_loss_cls, get_loss_schema_registry  # noqa
```

Comment:
General question: is this a normal practice in large Python codebases?

Reply:
It is for being able to import from a top-level module instead of a submodule. You can also control the public api using the `__all__` param, but we don't do that, so this is good enough.